packages/glibc.rb

```diff
@@ -7,28 +7,30 @@ class Glibc < Package
   compatibility 'all'
 
   depends_on 'gawk' => :build
-  depends_on 'libgd' => :build
   depends_on 'libidn2' => :build
   depends_on 'texinfo' => :build
   depends_on 'hashpipe' => :build
 
-  # enable-obsolete-rpc should NOT be included as a build option as
-  # this functionality is provided by libtirpc
+  no_env_options
+  conflicts_ok
 
-  case LIBC_VERSION
-  when '2.23'
-    version LIBC_VERSION
+  @LIBC_VERSION = LIBC_VERSION
+  # Uncomment following line to build a version of glibc different
+  # from the one ChromeOS ships with.
+  # @LIBC_VERSION = '2.33'
+  if @LIBC_VERSION == '2.23'.freeze
+    version '2.23-3'
     source_url 'https://ftpmirror.gnu.org/glibc/glibc-2.23.tar.xz'
     source_sha256 '94efeb00e4603c8546209cefb3e1a50a5315c86fa9b078b6fad758e187ce13e9'
 
     binary_url({
-      i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/glibc/2.23_i686/glibc-2.23-chromeos-i686.tar.xz'
+      i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/glibc/2.23-3_i686/glibc-2.23-3-chromeos-i686.tar.xz'
     })
     binary_sha256({
-      i686: '52145b65cb49c2751f69d4c46636f0685f2abb6685d8a080b71b2f091595a950'
+      i686: '3ee19cbb907eb219a2c1b02df6de1ca13b09b0d375101657d54a2485aacdc445'
     })
-  when '2.27'
-    version LIBC_VERSION
+  elsif @LIBC_VERSION == '2.27'
```

Comment:
Can constants be instance variables?  Why do we need the `@`?

Reply:
This will let you set the instance variable at the top so you can do cross-glibc-version builds.

Reply:
maybe these should all be lowercase?